letter.js

```diff
@@ -1,22 +0,0 @@
-//Constructor
-// "letter.js should control whether or not a letter appears as a "_" or as itself on-screen."
-var game = require("./game.js");
-var Word = require("./word.js");
-
-function Letter(letter){
-	this.letter = letter.toLowerCase(),
-	this.guessed = false
```

Comment:
excellent choice. just make sure to end with semicolons and not commas. we're not in an object setting here; we're in a function..